io/net/ethtool_test.py

```diff
@@ -124,24 +128,96 @@ def interface_link_status(self, interface):
                 return line.split()[-1]
         return ''
 
+    def ping_check_loss(self, interface, peer_ip,
+                        count, option=None, flood=False):
```

Comment:
this is a common function, keep it in utils

Reply:
yes good idea @abdhaleegit , please consider this to moving in utils

Reply:
@abdhaleegit @PraveenPenguin  Moved the function to util. Will share the PR for the same